src/popup/components/Tokens.vue

```diff
@@ -106,6 +107,14 @@ export default defineComponent({
       return `${AE_AVATAR_URL}${token.contractId}`;
     }
 
+    /**
+     * Some transactions are contract calls, but we still want to display the coin icon
+     * Thus if the transaction should display a coin, we use the protocol as contractId
+     */
+    function getDisplayTokenContractId(token: ITokenResolved) {
+      return token.assetType === ASSET_TYPES.coin ? props.protocol : token.contractId;
```

Comment:
This is actually coincidence that we use the same values for the protocol names and the contract IDs for the protocol coin. I think we should not mix those 2 concepts because this might blow if the value of `AE_CONTRACT_ID` or `PROTCOL.aeternity` will change.